### PR TITLE
flake8 fixes

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: django-stubs/db/models/__init__.pyi [hunk 1]
```diff
@@ -109,16 +109,6 @@ from .manager import BaseManager as BaseManager, Manager as Manager
 
 from . import lookups as lookups
 
-from .aggregates import (
```
Comment (sobolevn) [inline django-stubs/db/models/__init__.pyi:112]: This is required: https://github.com/django/django/blob/master/django/db/models/__init__.py#L3

**Review by sobolevn — CHANGES_REQUESTED** [review]

**Comment by sobolevn** [comment]
Thanks a lot!

File: django-stubs/db/models/__init__.pyi (hunk 1 above)
Comment (blueyed) [reply django-stubs/db/models/__init__.pyi:112]: It is above already (a bit more complete).
flake8 reported duplicate imports there.

**Comment by mkurnikov** [comment]
Good catch! Is it possible to add flake8 for `.pyi` checking via https://github.com/ambv/flake8-pyi ? (probably pin flake8 dependency too)

**Comment by blueyed** [comment]
Using flake8-pyi additionally sounds good to me.

**Comment by mkurnikov** [comment]
https://github.com/typeddjango/django-stubs/pull/186 for flake8-pyi

**Comment by blueyed** [comment]
Cool, rebased this PR here.

**Comment by mkurnikov** [comment]
How did you find out about those duplicates? I tried to look for this kind of warning in pyflakes/flake8, no luck.

**Comment by blueyed** [comment]
Via `flake8 **/*.pyi`.. ;)

Might make sense to only have a single config by now?
I think per-file-ignores then could be used to handle .py/.pyi differently if needed (and not provided via `flake8-pyi.ini`.

**Comment by mkurnikov** [comment]
> Might make sense to only have a single config by now?
I think per-file-ignores then could be used to handle .py/.pyi differently if needed (and not provided via flake8-pyi.ini.

I tried this approach first, but failed. Problem was that i want to ignore `F401` in `__init__.pyi` files, and check it everywhere else. 
If you'd be able to do it, it would be great.

**Comment by sobolevn** [comment]
@mkurnikov that's how it can be done: https://github.com/dry-python/returns/blob/master/setup.cfg#L41-L49

**Comment by blueyed** [comment]
Closing for now.

**Pull request closed without merging**